sql/analyzer/aliases.go

```diff
@@ -131,6 +127,10 @@ func getTableAliases(n sql.Node, scope *Scope) (TableAliases, error) {
 			panic("Table not resolved")
 		}
 
+		if opaque, ok := node.(sql.OpaqueNode); ok && opaque.Opaque() {
```

Comment:
Just to clarify: the reason for this change is because we were missing aliases from opaque `plan.SubqueryAlias` nodes, correct?

Reply:
getTableAliases was ignoring SubqueryAlias nodes. I'm not sure why this never caused errors elsewhere, or it is possible that this change may cause an error in the future. I can't rationalize why subqueryAliases should be excluded from the list of named nodes in a scope, so going this way seems preferable as a default.